### PR TITLE
Solver project url job

### PR DESCRIPTION
**Pull request by bjoernh2000**
## Related Issues and Dependencies

thoth-station/core#286

## Description

.yaml file to create cronjob for solver-project-url-job

### CODE REVIEW

File: solver-project-url-job/base/cronjob.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+kind: CronJob
+metadata:
+  name: solver-project-url-job
+  labels:
+    app: thoth
+    component: solver-project-url-job
+spec:
+  schedule: "@daily"
+  successfulJobsHistoryLimit: 2
+  failedJobsHistoryLimit: 4
```
Comment (harshad16) [inline solver-project-url-job/base/cronjob.yaml:12]:
```suggestion
  successfulJobsHistoryLimit: 1
  failedJobsHistoryLimit: 1
```
Lets keep these just one

File: solver-project-url-job/base/cronjob.yaml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                    secretKeyRef:
+                      name: ceph
+                      key: secret-key
+              resources:
+                requests:
+                  memory: "256Mi"
+                  cpu: "1"
+                limits:
+                  memory: "256Mi"
+                  cpu: "1"
```
Comment (harshad16) [inline solver-project-url-job/base/cronjob.yaml:81]:
```suggestion
                  cpu: "100m"
                limits:
                  memory: "256Mi"
                  cpu: "300m"
```
lets start with this config

**Review by harshad16 — COMMENTED** [review]
/lgtm
just few minor comments :+1:

**Review by harshad16 — COMMENTED** [review]
can you follow the other cronjob, like graph-refresh 
and create the overlays, add kustomization.yaml and other files too :+1:

File: solver-project-url-job/base/cronjob.yaml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    spec:
+      template:
+        metadata:
+          labels:
+            app: thoth
+            component: solver-project-url-job
+        spec:
+          restartPolicy: OnFailure
+          containers:
+            - image: "quay.io/thoth-station/solver-project-url-job:v0.1.5"
```
Comment (harshad16) [inline solver-project-url-job/base/cronjob.yaml:25]: as now we have imagestream, lets use that
```suggestion
            - image: "solver-project-url-job:latest"
```

Comment (bjoernh2000) [reply solver-project-url-job/base/cronjob.yaml:25]: ok sounds good!

**Review by harshad16 — COMMENTED** [review]
lgtm
can you please rebase your commit to one.

**Review by harshad16 — APPROVED** [review]
/lgtm
thanks :100:

**Pull request merged**